Tests/powershell-helpers.ps1

```diff
@@ -37,18 +37,29 @@ function Test-AppExists($appName) {
   return $null -ne $command
 }
 
-function Test-PowershellModuleInstalled($moduleName, $version) {
-  $command = Get-Module $moduleName -listavailable
+function Import-PowerShellModule ($Name, $MinimumVersion) {
+  Write-Output "Checking version for $moduleName..."
+  $command = Get-Module $Name -listavailable
   if ($null -eq $command) {
-    write-host "Please install $($moduleName): Install-Module -Name $moduleName -Force" -foregroundcolor red
+    write-host "Please install $($Name): Install-Module -Name $Name -Force" -foregroundcolor red
     exit 1
   }
-  if ($null -ne $version) {
-    if ($command.Version -lt [System.Version]::Parse($version)) {
-      write-host "Please install $($moduleName) $version or higher (you have version $($command.Version)): Update-Module -Name $moduleName -Force" -foregroundcolor red
+
+  if ($command.Count -gt 1)
+  {
+    $command = $command | Sort-Object -Property Version -Descending | Select-Object -First 1
```

Comment:
Will this sort as `string` or `system::version`?

Reply:
Property Version has the same properties (Major, Minor, etc...), however, object type comparison returns false.  Casting the property to a System.Version object works so I could iterate through the returned objects casting the Version property to System.Version and add it to an array, then use the same Sort-Object command to select the highest version.  Thoughts?

Reply:
Just tried:
```
get-module -ListAvailable | Sort-Object -Property Version -Descending
```
and it listed everything in proper order, so I think you're fine.